docs/FAQ.md

```diff
@@ -39,14 +39,14 @@ links for the latest release.
 On startup the RAPIDS Accelerator will log a warning message on the Spark driver showing the
 version with a message that looks something like this:
 ```
-21/04/14 22:14:55 WARN SQLExecPlugin: RAPIDS Accelerator 0.5.0 using cudf 0.19. To disable GPU support set `spark.rapids.sql.enabled` to false
+21/04/14 22:14:55 WARN SQLExecPlugin: RAPIDS Accelerator 0.5.0 using cudf 0.19.1. To disable GPU support set `spark.rapids.sql.enabled` to false
 ```
 
 The full RAPIDS Accelerator and cudf build properties are logged at `INFO` level in the
 Spark driver and executor logs with messages that are similar to the following:
 ```
-21/04/14 17:20:20 INFO RapidsExecutorPlugin: RAPIDS Accelerator build: {version=0.5.0-SNAPSHOT, user=jlowe, url=, date=2021-04-14T22:12:14Z, revision=79a5cf8acd615587b2c7835072b0d8b0d4604f8b, cudf_version=0.19-SNAPSHOT, branch=branch-0.5}
-21/04/14 17:20:20 INFO RapidsExecutorPlugin: cudf build: {version=0.19-SNAPSHOT, user=, date=2021-04-13T08:42:40Z, revision=a5d2407b93de444a6a7faf9db4b7dbf4ecbfe9ed, branch=HEAD}
+21/04/14 17:20:20 INFO RapidsExecutorPlugin: RAPIDS Accelerator build: {version=0.5.0-SNAPSHOT, user=jlowe, url=, date=2021-04-14T22:12:14Z, revision=79a5cf8acd615587b2c7835072b0d8b0d4604f8b, cudf_version=0.19.1, branch=branch-0.5}
+21/04/14 17:20:20 INFO RapidsExecutorPlugin: cudf build: {version=0.19.1, user=, date=2021-04-13T08:42:40Z, revision=a5d2407b93de444a6a7faf9db4b7dbf4ecbfe9ed, branch=HEAD}
```

Comment:
Note that if we manually update these lines then the reported revision will be inconsistent with the stated version.  It would be better to generate new output lines from an actual run against 0.19.1 with an updated revision commit hash, date, etc.

Reply:
So should we make them an older version that is correct, because it is just an example? or do we plan on trying to maintain this going forward for every release because a sed script will not be enough to keep it updated?

Reply:
I'm cool with leaving it just as an example that isn't updated, but I don't think we should hand-patch just portions of it.

Reply:
I agree

Reply:
sure,  I will leave it as just an example here